tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -285,6 +292,18 @@ def comparison_rule_below(self):
         self.trigger.update(alert_threshold=50)
         return rule
 
+    @cached_property
+    def dynamic_rule(self):
+        rule = self.rule
+        rule.update(
+            detection_type=AlertRuleDetectionType.DYNAMIC,
+            sensitivity=AlertRuleSensitivity.HIGH,
+            seasonality=AlertRuleSeasonality.AUTO,
+        )
+        # dynamic alert rules have a threshold of 0.0
+        self.trigger.update(alert_threshold=0)
```

Comment:
🤔 

not quite sure, but firing updates within the cached property seems odd to me...
Are you seeing what you expect to from these tests?